be/extmap.c

```diff
@@ -917,37 +1053,60 @@ static int emap_it_open(struct m0_be_emap_cursor *it)
 	rc = op->bo_u.u_btree.t_rc;
 	if (rc == 0) {
 		m0_be_btree_cursor_kv_get(&it->ec_cursor, &keybuf, &recbuf);
+
+		/* Key operation */
 		key = keybuf.b_addr;
-		rec = recbuf.b_addr;
 		it->ec_key = *key;
+
+		/* Record operation */
+		if (it->ec_recbuf.b_addr != NULL) {
+			m0_buf_free(&it->ec_recbuf);
+		}
+
+		/* Layout/format of emap-record (if checksum is present) which
+		 * gets written:
```

Comment:
[* style *] use tab.
same for next two lines.

Reply:
Done.